feat: Add limitations for nested virtualization (#171)
citynetwork/docs: docs/reference/limitations/openstack.md

  > You can use `openstack project list`, but this will only list the project(s) that your user account has access to.
* `openstack domain <subcommand>`

## Nova

### Nested virtualization

> Running nested virtualization is only supported for Nova instances running Linux as the guest operating system.

Reviewer: Nova instances act like hosts, right? If so, then I would suggest this change:
```suggestion
Nested virtualization only works in Nova instances running Linux.
```

Author: Yes, can do.

> The guest must run a [Linux kernel](https://www.kernel.org/) of version 5.0 or later, and [QEMU/KVM](https://www.qemu.org/) 4.1 or later.

Reviewer: Also, if I understand correctly, I believe this is more accurate:
 ```suggestion
Guests must be created with [QEMU/KVM](https://www.qemu.org/) 4.1 or later, and should run a [Linux kernel](https://www.kernel.org) of version 5.0 or later.
```

Author: I rephrased this slightly, and added information on what base images should be good to use for this purpose.

Reviewer: Very nice!

> Furthermore, you must ensure that the Nova instance [passes the `pcid` CPU feature flag](https://qemu-project.gitlab.io/qemu/system/qemu-cpu-models.html#important-cpu-features-for-intel-x86-hosts) to the nested guest.

Reviewer: ```suggestion
Furthermore, you must ensure that the Nova instance [passes the `pcid` CPU feature flag](https://qemu-project.gitlab.io/qemu/system/qemu-cpu-models.html#important-cpu-features-for-intel-x86-hosts) to nested guests.
```

Author: Done.
